docs/source/api/v4/service_categories_name.rst

```diff
@@ -0,0 +1,133 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-service-categories-name:
+
+***********************************
+``service_categories/{{name}}``
+***********************************
+
+``PUT``
+========
+Update a service category.
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Permissions Required: SERVICE-CATEGORY:UPDATE, SERVICE-CATEGORY:READ
+:Response Type:  Object
+
+Request Structure
+-----------------
+:name:        This :term:`Service Category`'s name
```

Comment:
This should be in a Path Parameters table, field lists are for request (and response) bodies

Reply:
It would need both.